src/azure-cli/azure/cli/command_modules/identity/aaz/latest/identity/__cmd_group.py

```diff
@@ -0,0 +1,23 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command_group(
+    "identity",
+)
+class __CMDGroup(AAZCommandGroup):
+    """Managed Identity Command Group
```

Comment:
It is better to keep consistent with the previous help message of the command group. Besides, please remove redundant code in _help.py file.
```suggestion
    """Managed Identities
```